heka/files/lua/common/gse.lua

```diff
@@ -36,10 +36,10 @@ local ordered_clusters = {}
 
```

Comment:
I would really like to have Lua unittests. What about the feature branch **mkxx** on legacy repository ?